Exiled.API/Features/Log.cs

```diff
@@ -7,102 +7,138 @@
 
 namespace Exiled.API.Features
 {
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
     using System.Reflection;
 
-    using Discord;
+    using Interfaces;
 
     /// <summary>
     /// A set of tools to print messages on the server console.
     /// </summary>
     public static class Log
     {
+        private static Dictionary<Assembly, bool> knownDebugValues = new();
+
         /// <summary>
-        /// Sends a <see cref="LogLevel.Info"/> level messages to the game console.
+        /// Sends a <see cref="Discord.LogLevel.Info"/> level messages to the game console.
         /// </summary>
         /// <param name="message">The message to be sent.</param>
-        public static void Info(object message) => Send($"[{Assembly.GetCallingAssembly().GetName().Name}] {message}", LogLevel.Info, System.ConsoleColor.Cyan);
+        public static void Info(object message) => Send($"[{Assembly.GetCallingAssembly().GetName().Name}] {message}", Discord.LogLevel.Info, System.ConsoleColor.Cyan);
 
         /// <summary>
-        /// Sends a <see cref="LogLevel.Info"/> level messages to the game console.
+        /// Sends a <see cref="Discord.LogLevel.Info"/> level messages to the game console.
         /// </summary>
         /// <param name="message">The message to be sent.</param>
-        public static void Info(string message) => Send($"[{Assembly.GetCallingAssembly().GetName().Name}] {message}", LogLevel.Info, System.ConsoleColor.Cyan);
+        public static void Info(string message) => Send($"[{Assembly.GetCallingAssembly().GetName().Name}] {message}", Discord.LogLevel.Info, System.ConsoleColor.Cyan);
 
         /// <summary>
-        /// Sends a <see cref="LogLevel.Debug"/> level messages to the game console.
+        /// Sends a <see cref="Discord.LogLevel.Debug"/> level messages to the game console.
         /// Server must have exiled_debug config enabled.
         /// </summary>
         /// <param name="message">The message to be sent.</param>
-        /// <param name="canBeSent">Indicates whether the log can be sent or not.</param>
-        public static void Debug(object message, bool canBeSent = true)
+        public static void Debug(object message)
         {
-            if (canBeSent)
-                Send($"[{Assembly.GetCallingAssembly().GetName().Name}] {message}", LogLevel.Debug, System.ConsoleColor.Green);
+#if !DEBUG
```

Comment:
We shouldn't have three methods with the exact same code except for a string, we should only have one method which cointains all the code and then call it with the other two.